Utils/API/server/lib/dkb/api/exceptions.py

```diff
@@ -0,0 +1,135 @@
+"""
+Exception definitions for DKB API server.
+"""
+
+import inspect
+
+from . import ADDR
+
+
+class DkbApiException(Exception):
+    """ Base exception for DKB API module. """
+    code = 500
+    details = 'Unknown error'
+
+
+class DkbApiNotImplemented(DkbApiException, NotImplementedError):
+    """ Exception for not implemented functional. """
+    code = 501
+    details = "What you are looking for is not implemented yet."
+
+
+class NotFoundException(DkbApiException):
+    """ Base exception for "smth not found" errors. """
+    code = 400
+    details = "For list of available methods and (sub)categories:" \
+              " %s/<category>/info" % ADDR
+
+
+class CategoryNotFound(DkbApiException):
+    """ Exception indicating that category not found. """
```

Comment:
This was probably meant to be "... that category IS not found.".

Reply:
Fixed: #370.